symengine/functions.cpp

```diff
@@ -165,8 +165,7 @@ bool could_extract_minus(const Basic &arg)
     if (is_a_Number(arg)) {
         if (down_cast<const Number &>(arg).is_negative()) {
             return true;
-            // TODO: see #915
-        } else if (is_a<Complex>(arg)) {
+        } else if (is_a_Complex(arg)) {
             const Complex &c = down_cast<const Complex &>(arg);
```

Comment:
I meant that I'll update `is_a_Complex()` in a way similar to `is_a_Number()` function.